src/rgw/rgw_tools.cc

```diff
@@ -18,10 +18,10 @@ static std::map<std::string, std::string>* ext_mime_map;
 
 void parse_mime_map_line(const char *start, const char *end)
 {
-  char line[end - start + 1];
-  strncpy(line, start, end - start);
+  std::vector<char> line(end - start + 1);
+  strncpy(line.data(), start, end - start);
```

Comment:
Might as well do this in the declaration, along the lines of:
vector<char> line(start, end - start);
or
string line(start, end - start);
or EVEN:
string_view line { start, end - start };
...if we don't actually need a copy.

Reply:
We do unless we want to rewrite it not to use `strsep`. I'm not opposed, but I also don't want to do it right now.